### PR TITLE
Add publish workflow

### PR DESCRIPTION
**Pull request by FynnBe**

### CODE REVIEW

File: .github/workflows/backup.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  S3_BUCKET: ${{vars.S3_BUCKET}}
+  S3_FOLDER: ${{vars.S3_FOLDER}}
+  ZENODO_URL: ${{vars.ZENODO_URL}}
+  S3_ACCESS_KEY_ID: ${{secrets.S3_ACCESS_KEY_ID}}
+  S3_SECRET_ACCESS_KEY: ${{secrets.S3_SECRET_ACCESS_KEY}}
+  ZENODO_API_ACCESS_TOKEN: ${{secrets.ZENODO_API_ACCESS_TOKEN}}
+
+jobs:
+  call:
+    uses: ./.github/workflows/publish_call.yaml
```
Comment (jmetz) [inline .github/workflows/backup.yaml:20]: Should this be `backup_call` ?

**Comment by jmetz** [comment]
@FynnBe - could you share / let me know what local testing you're using for workflows? Are you using the local actions runner that we discussed a short while ago still or more just doing local testing of the python scripts called by the workflows?

**Comment by oeway** [comment]
Great to see the work here! Thank you both!

File: .github/workflows/backup.yaml (hunk 1 above)
Comment (FynnBe) [reply .github/workflows/backup.yaml:20]: YES!

**Comment by FynnBe** [comment]
> @FynnBe - could you share / let me know what local testing you're using for workflows? Are you using the local actions runner that we discussed a short while ago still or more just doing local testing of the python scripts called by the workflows?

I now only test with pytest locally and try to have the pytests map all processes. 
Also I try to keep the workflows very light, e.g. call only one script.
And then testing online (test.yaml) workflow isn't so painful when pytests pass locally

**Pull request merged**